src/ContaCorrente.java

```diff
@@ -0,0 +1,21 @@
+public class ContaCorrente extends Conta implements ITaxas{
+    public ContaCorrente(int numero, Cliente dono, double saldo) {
+        super(numero, dono, saldo);
+    }
+
+    @Override
+    public void setLimite(double limite) {
+        if (limite < -100) {
+            this.limite = -100; // Correção aqui para definir o limite mínimo como -100
+            System.out.println("Limite não pode ser inferior a -100R$!");
+        } else {
+            this.limite = limite;
+            System.out.println("Limite dentro das regras do banco! :)");
+        }
+    }
+
+    @Override
+    public double calculaTaxas() {
+        return 0;

```

Comment:
O exercício pede para implementar a interface na classe ContaCorrente e taxar em 10 reais se cliente for uma pessoa física, ou 20 reais se pessoa jurídica. Isso deve ser feito através do polimorfismo, ou seja, tratar objetos de diferentes classes de maneira uniforme, sem precisar saber o tipo específico do objeto.
Assim: return this.getDono().calcularTaxas().